src/compiler/builder.ts

```diff
@@ -546,70 +549,59 @@ namespace ts {
         }
 
         Debug.assert(!!state.currentAffectedFilesExportedModulesMap);
+
         const seenFileAndExportsOfFile = new Set<string>();

```

Comment:
Separate PR: @sheetalkamat can we lift this out a couple levels so we don't revisit the same paths repeatedly for different "affected" files?

Reply:
Or maybe it belongs on the state?

Reply:
To share it across more than one affected file, it would have to be on the state.  I'm going to try updating it in parallel with `seenAffectedFiles`.

Reply:
Assuming I've done it correctly, it saves about 10% of total time.  Nothing to complain about, but much less than this change.